qiita_pet/support_files/doc/source/processingdata/deblur_2021.09.rst

```diff
@@ -0,0 +1,42 @@
+deblur version 2021.09
+======================
+
+The deblur version 2021.09 addresses a bug with the fragment insertion parsing and
+cache that ignored some fragments for getting an accurate placement in the tree. In
+summary, in some occasions SEPP will return multiple fragments in a single entry; which
+was unexpected by the qp-deblur plugin parser, which assumed only one entry - the
+extra features will be seen as missing by the plugin and that information was
+sent and stored in the cache provided by Qiita, then propagated to future studies and
+meta-analyses.
+
+This bug was resolved in this `pull request <https://github.com/qiita-spots/qp-deblur/pull/60>`__.
+
+Sample counts implications
+--------------------------
+
+At the time of writing Qiita had 978,052 16S deblured private or pubic samples.
+In the figure below, we have at different trimming lengths how many samples we recover
+based on the minimum number of sequences per sample - this is an important consideration
+as we normally need to remove samples below a given threshold for beta diversity
+calculations (via rarefactoin) or differential abundance testing.
```

Comment:
this only applies, if rarefaction was based off faith PD, all other metrics should not have been affected, iff you did not additionally filter the deblur tables provided by Qiita down to those features contained in the insertion tree file (which comes together with the biom file in qiita)

Reply:
Good point, but this is true _only_ for the full/all table not for the reference-hit because the table needs to be filtered to match what's in the tree, which is automatically done via the meta-analysis construction in Qiita. Do you have any specific text suggestions to cover this ?

Reply:
I tried to add some more information about this at the end of the intro paragraph, please let me know if that works.